src/ComputationContainer/Job/JobStatus.hpp

```diff
@@ -3,8 +3,8 @@
 #include <boost/format.hpp>
 #include <optional>
 #include <stdexcept>
-
-#include "Client/ComputationToDbGate/Client.hpp"
+;
```

Comment:
Cloud you remove semi-colon?

Reply:
I remove it in ac007d5bf85cb16e4942e0cda18aaa3aa71af9b8!

Reply:
Thank you!